.github/workflows/compilation.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         commit-filter: '[skip ci]'
     - name: Install dependencies
       run: |
-        module load python/3.11.3
+        module load python
```

Comment:
This clashes with what is now on master. As discussed elsewhere, I think we should specify the version of Python explicitly so that we're in control of changing it.

Reply:
Ah yes, I was expecting this clash, reverted to current master